src/Entities/Note.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Entities;
+
+use Rebilly\Http\Exception\UnprocessableEntityException;
+use Rebilly\Rest\Entity;
+
+/**
+ * Class Note
+ *
+ * ```json
+ * {
+ *   "id"
+ *   "relatedType"
+ *   "relatedId"
+ *   "content"
+ *   "archived"
+ *   "createdBy"
+ *   "createdTime"
+ *   "updatedTime"
+ *   "archivedTime"
+ * }
+ * ```
+ *
+ * @author Arman Tuyakbayev <arman.tuyakbayev@rebilly.com>
+ * @version 0.1
+ */
+final class Note extends Entity
+{
+    const RELATED_TYPE_CUSTOMER = 'customer';
+    const RELATED_TYPE_WEBSITE = 'website';
+
+    public $allowableRelatedTypes = [
+        self::RELATED_TYPE_CUSTOMER,
+        self::RELATED_TYPE_WEBSITE,
+    ];
+
+    /**
+     * @return string
+     */
+    public function getCreatedTime()
+    {
+        return $this->getAttribute('createdTime');
+    }
+
+    /**
+     * @return string
+     */
+    public function getArchivedTime()
+    {
+        return $this->getAttribute('archivedTime');
+    }
+
+    /**
+     * @return string
+     */
+    public function getRelatedType()
+    {
+        return $this->getAttribute('relatedType');
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function setRelatedType($value)
+    {
+        if (!in_array($value, $this->allowableRelatedTypes)) {
+            throw new UnprocessableEntityException('Wrong relatedType');
```

Comment:
@XaosSintez please look to [this example](https://github.com/Rebilly/rebilly-php/blob/master/src/Entities/Blacklist.php#L61)